content/00.index.md

```diff
@@ -44,7 +44,7 @@ These SDKs, available in various programming languages, streamline the process o
     and on-chain governance.
 
 ::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
-We strongly recommend reading [**the introduction**](/sdk#what-is-zksync-era-sdks) before moving on to the rest of the
+We strongly recommend reading [**the introduction**](#what-is-zksync-era-sdks) before moving on to the rest of the
```

Comment:
I know it was here before but is this link meant to go to the top of the same page

Reply:
It is, it was written by the technical writer.